Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "cramjam"
-version = "2.6.1"
+version = "2.6.2"
 authors = ["Miles Granger <miles59923@gmail.com>"]
 edition = "2018"
 license-file = "LICENSE"
```

Since it's the standard MIT license, maybe just use SPDX license identifier instead.
```suggestion
license = "MIT"
```